packages/config/config/devices/0x0129/cp2-5-3_1.78.json

```diff
@@ -53,18 +52,18 @@
 			"allowManualEntry": false,
 			"options": [
 				{
-					"label": "Auto Relock On",
+					"label": "Enable",
 					"value": -128
```

Comment:
We could make this `128` without the minus if we set `unsigned: true`, but I don't think this is necessary

Reply:
So you add "unsigned": true and change the -128 to 128?